pkg/config/config.go

```diff
@@ -186,6 +192,16 @@ func NewDefaultConfig() *Config {
 				},
 			},
 		},
+		WaiterContainerTemplate: corev1.Container{
+			Image: waiterDefaultImage,
+			Args: []string{
+				"start",
+			},
```

Comment:
Please include the command. If we do not include it, [Tekton would try to pull the image configuration to determine the entrypoint](https://github.com/tektoncd/pipeline/blob/main/pkg/pod/entrypoint_lookup_impl.go) so that it can rewrite it. That's a small performance hit for the first run and can be a mess if the image cannot be pull anonymously. I think if it is loaded into kind (kind.local), this also would not work.

Reply:
Thanks for the detailed explanation, I've added the `Command` attribute too. Please consider.